src/App.jsx

```diff
@@ -19,6 +19,24 @@ class App extends React.Component {
   }
 
   render() {
+    var lock = new Auth0Lock('KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH', 'loom.auth0.com');
+    var btn_login = document.getElementById('btn-login');
```

Comment:
I'm not yet an expert, but at the least, we'll have to have a `<Button />` component with an `onClick` listener. to expand on that, it'd probably make sense to have some kind of "login" component or container (these are somehow different), which contains all this lock logic as well as the `<Button />`